libensemble/__init__.py

```diff
@@ -9,3 +9,4 @@
 __credits__ = 'Argonne National Laboratory'
 
 from libensemble import libE_logger
+from .libE import libE, comms_abort
```

Comment:
I've never seen "from .package import name" with the dot. Is that intentional?

Reply:
Yes, this is intentional. It imports from libE in the same directory as the __init__

Reply:
This line makes ``from libensemble import libE`` possible.